packages/generator-langium/src/index.ts

```diff
@@ -129,15 +141,35 @@ class LangiumGenerator extends Generator {
                 }
             );
         }
+
+        if (this.answers.targetEnvironment === 'web') {
```

Comment:
The other regular target is always created. So, web is always optional or do web want an exclusive or (XOR))?

Reply:
Most of the files generated are shared between `vscode` and `web` environment. 
I didn't want to duplicate the vscode files (and increase maintenance cost) for the web configuration. Instead I just overwrite what is needed.

Reply:
Ok, makes sense. Thanks for explaining.